pypika/queries.py

```diff
@@ -889,12 +898,16 @@ def __add__(self, other):
     def __mul__(self, other):
         return self.union_all(other)
 
+    @builder
+    def slice(self, slice):
+        self._offset = slice.start
+        self._limit = slice.stop
```

Comment:
Shouldn't the limit be `stop-start`? I could be wrong of course I didn't look into the caller or the usage.

Reply:
I guess since it worked like that before beneath, I'm sure it's still correct.

Reply:
didn't think about it, was just moving some old code around. Sounds like it's worth looking into.